pkg/fsutils/idtools.go

```diff
@@ -1,4 +1,4 @@
-package idtools
+package fsutils
```

Comment:
I think some of this file needs to be split up.

All the Mkdir... would be fsutils, but the ID stuff probably makes more sense in system/user/

The file stiff can stay in fsutil